sonic_package_manager/service_creator/creator.py

```diff
@@ -283,6 +283,7 @@ def generate_container_mgmt(self, package: Package):
             'docker_container_name': name,
             'docker_image_id': image_id,
             'docker_image_name': package.entry.repository,
+            'docker_image_version': package.entry.default_reference,
```

Comment:
```default_reference``` points to default image not the one that is installed.
In case user upgrades - it no longer points to a correct version